system-int-test/src/cmr/system_int_test/utils/variable_util.clj

```diff
@@ -319,3 +319,27 @@
   (search/process-response
    (transmit-variable/search-for-variables (s/context) params {:raw? true
                                                                :http-options {:accept :json}})))
+
+(defn- get-single-variable-from-umm-json
+  "Returns a single variable from a UMM JSON response. Returns nil if the provided concept-id and
+  revision-id are not found."
+  [umm-json-response concept-id revision-id]
+  (let [variables (filter (fn [item]
```

Comment:
I'd like to see the anonymous filter function here put into a named private function above ...

Reply:
Done.